app/src/main/java/com/vero/sibs/ui/registration/RegistrationFragment.kt

```diff
@@ -26,18 +27,26 @@ class RegistrationFragment : Fragment(R.layout.fragment_registration) {
         binding = DataBindingUtil.inflate(inflater, R.layout.fragment_registration, container,false)
         binding.lifecycleOwner = this
         binding.viewModel = viewModel
+
         return binding.root
     }
 
+    override fun onResume() {
+        super.onResume()
+        (requireActivity() as AppCompatActivity).supportActionBar?.hide()
```

Comment:
I see the line of code a second time in your PR. For sure it can be replaced with an Extension. The DRY principle is broken